config/config.go

```diff
@@ -9,13 +11,15 @@ type Config struct {
 	FilterAPIURL         string `envconfig:"FILTER_API_URL"`
 	DatasetAPIURL        string `envconfig:"DATASET_API_URL"`
 	HierarchyAPIURL      string `envconfig:"HIERARCHY_API_URL"`
-	DatasetAPIAuthToken  string `envconfig:"DATASET_API_AUTH_TOKEN"`
-	FilterAPIAuthToken   string `envconfig:"FILTER_API_AUTH_TOKEN"`
-	SearchAPIAuthToken   string `envconfig:"SEARCH_API_AUTH_TOKEN"`
+	DatasetAPIAuthToken  string `envconfig:"DATASET_API_AUTH_TOKEN" json:"-"`
+	FilterAPIAuthToken   string `envconfig:"FILTER_API_AUTH_TOKEN"  json:"-"`
+	SearchAPIAuthToken   string `envconfig:"SEARCH_API_AUTH_TOKEN"  json:"-"`
 	SearchAPIURL         string `envconfig:"SEARCH_API_URL"`
 	DownloadServiceURL   string `envconfig:"DOWNLOAD_SERVICE_URL"`
 	EnableDatasetPreview bool   `envconfig:"ENABLE_DATASET_PREVIEW"`
 	EnableLoop11         bool   `envconfig:"ENABLE_LOOP11"`
+	Interval             string `envconfig:"HEALTHCHECK_INTERVAL"`
```

Comment:
We can use type time.Duration directly here:
```
HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
```

Reply:
That will only allow nanosecond value, prefer to use `time.ParseDuration` on 10m or 3000ms instead of making Interval 1000000000 for 1 second value which would be 1s

Reply:
See next commit for changes